data.sql

```diff
@@ -11,3 +11,24 @@ VALUES ('Pikachu', '2021-01-07', 15.04, false, 1);
 
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts )
 VALUES ('Devimon', '2017-05-12', 11, true, 0);
+
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts )
+VALUES ('Charmander', '2020-02-08', 11, false, 0);
+
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
+VALUES ('Plantmon', '2021-11-15', 5.7, true, 2);
+
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
+VALUES ('Squirtle', '1993-04-02', 12.13, false, 3);
+
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
+VALUES ('Angemon', '2005-06-12', 45, true, 1);
```

Comment:
- [ ] According to the project requirements, all these pokemon weights should be negative values.
![image](https://user-images.githubusercontent.com/45460192/213654077-008d7c19-b8fc-41ac-9b99-28bbbf8b14c1.png)